### PR TITLE
[Chinese Translation] FAQ and some titles

### PR DESCRIPTION
**Pull request by Bisstocuz**

### CODE REVIEW

File: docs/content/schinese/Asset Management/faq.md [hunk 1]
```diff
@@ -1,42 +1,49 @@
 +++
-title = "Frequently Asked Questions"
+title = "常见问题解答"
 weight = 1
 +++
 
-##### How to install the tools?
```
Comment (WuZhuoran) [inline docs/content/schinese/Asset Management/faq.md:6]: 建议把标题的 级数 改成 ##

File: docs/content/schinese/Asset Management/faq.md [hunk 2]
```diff
@@ -1,42 +1,49 @@
...
 
-##### I have mods with DLL’s and my modding tools does not launch.
-If you have any mods with DLL’s enabled, be sure to copy them to the Modules\MOD_NAME\bin\Win64_Shipping_wEditor folder as well.
+##### 我无法添加新模型、材质等
 
-##### My modding tool does not launch.
-Be sure that Visual Studio 2013 Redist x64 is installed in your machine. Also, check whether the version numbers for the base game and modding tools match. If all of the above checks does not work, you can create a new topic in the “Bug & Crashes” section of the modding forums(https://forums.taleworlds.com/index.php?forums/bug-crash-reports.784/). 
\ No newline at end of file
+你无法直接向游戏本体添加资源，你需要在你自己的模组里添加它们。首先，你需要新建一个模组：http://docs.modding.bannerlord.com/zh_cn/asset-management/quickguide_create_a_mod/
+更多关于添加新的资产的信息请参阅： http://docs.modding.bannerlord.com/zh_cn/asset-management/asset-types/overriding_assets/
```
Comment (WuZhuoran) [inline docs/content/schinese/Asset Management/faq.md:25]: It is better to use `{{ref "the file path"}}`

File: docs/content/schinese/Asset Management/faq.md [hunk 3]
```diff
@@ -1,42 +1,49 @@
...
+
+##### 可供参考的示例场景？
+
+empire_village_003<br>
```
Comment (WuZhuoran) [inline docs/content/schinese/Asset Management/faq.md:29]: Not need to <br>    
Use 2 space at the end of each line.

File: docs/content/schinese/Asset Management/faq.md (hunk 1 above)
Comment (Bisstocuz) [reply docs/content/schinese/Asset Management/faq.md:6]: Follow original files for now.

**Review by WuZhuoran — APPROVED** [review]
LGTM

**Review by OzgurEraslan — APPROVED** [review]

**Pull request merged**